commit 8f616289d892950f127a1df87bcc66ff9dbdb11a
Author: Laurent Montel <montel@kde.org>
Date:   Fri Aug 27 22:06:30 2010 +0000

    backport: fix compile with qt4.7
    
    svn path=/branches/KDE/4.5/kdebase/workspace/; revision=1168925

--- a/plasma/generic/applets/battery/battery.h
+++ b/plasma/generic/applets/battery/battery.h
@@ -42,7 +42,7 @@
 class Battery : public Plasma::PopupApplet
 {
     Q_OBJECT
-    Q_PROPERTY(qreal labelAlpha READ labelAlpha WRITE setLabelAlpha NOTIFY labelAlphaChanged)
+    Q_PROPERTY(qreal labelAlpha READ labelAlpha WRITE setLabelAlpha )
     Q_PROPERTY(qreal acAlpha READ acAlpha WRITE setAcAlpha)
 
     public:
